<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>466</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>553</width>
    <height>466</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>553</width>
    <height>466</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Registro de Asistencias</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidgetConAsistencia">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>421</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>SOCIO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>APELLIDO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NOMBRE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ACTIVIDAD</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PROMO</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
